### DISCUSSION TITLE
I used tons of photos and my mesh is still awful. Why?!

### DISCUSSION THREAD

**Discussion by georgemoise1001** [alicevision/Meshroom#2201] [Q&A] [answered]
Hey there. I'm trying to do a photoscan of my friend. I took tons of photos, thinking I wouldn't have any problem at all, but the mesh that came back was awful. Here's a video of all the stills:

https://drive.google.com/file/d/1HM2tTnYiYeWy2xh6yxqN4wKMBo6IrV0s/view?usp=sharing

All I care about is the guy's head. Once I deleted all the room geo in the background in Blender, this is the mesh I got back:

https://drive.google.com/file/d/1CTWd439is0KQWvMZZG_aG7v88fUQ5Ig-/view?usp=sharing

I don't know what went wrong. Anyone have any thoughts??

thank you!!

George
www.georgemoise.com

**Comment by natowi** [comment, marked as answer]
Here are a few things that negatively affect the results: bad lighting (over/under exposure), motion blur, slight movements of your friend
Capturing people is a challenge even with a good setup. Image quality and good capturing setup is key.
Let your friend lean on some support structure to limit movement and do close up images of the face.

Check out the tutorials here: https://github.com/alicevision/Meshroom/wiki/Tutorials

**Reply by georgemoise1001** [reply]
thx for the feedback! What I if I isolated just his head in all the photos? Like in photoshop? And cropped the images to be from the neck up? Would that get me a better result?

**Reply by natowi** [reply]
No, don´t crop images. The overall quality of the images has much room for improvements. Make yourself familiar with the special requirements for capturing images for photogrammetry. Here you can see an ideal example of a professional photo shooting: https://triplegangers.com/blog/cat/technology/post/free-scan-sample/. ~25 good images can result in a great reconstruction. They use a camera rig, but handheld cameras do work, too.

This is how I do a reconstruction of a head: even lighting, back of the head resting against a wall. Shooting overlapping images from front, up and down, ~25. Backside of the head can be fixed later in 3d software as hair usually does not reconstruct well and needs to be replaced anyway.

**Reply by georgemoise1001** [reply]
thank you!
